tests/init.js

```diff
@@ -0,0 +1,28 @@
+define('tests/init',
+    ['core/init', 'core/nunjucks', 'core/router', 'core/urls'],
+    function(init, nunjucks, router, urls) {
+
+    describe('init', function() {
+        it('returns a resolving promise', function(done) {
+            init.promise.done(function() {
+                assert.ok(window.navigator.l10n);
+                done();
+            });
+        });
+
+        it('sets up helpers', function() {
+            assert.ok(nunjucks.require('globals').url);
```

Comment:
Don't think it worked for me. Or maybe something else was going wrong. I kept the ok's for now